draft-demarco-oauth-status-assertions.md

```diff
@@ -234,9 +234,10 @@ referenced Digital Credential;
 - MUST be timestamped with its issuance datetime,
 using a timestamp which is at or after the time of
 Digital Credential issuance which it refers;
-- MUST contain the expiration datetime after which both
-the Status Assertion and the Digital Credential it refers
-MUST NOT be considered valid anymore. The expiration datetime MUST be
+- MUST contain the expiration datetime after which
+the Status Assertion MUST NOT be considered valid anymore,
+and the Digital Credential it refers
+SHOULD NOT be considered valid anymore. The expiration datetime MUST be
 superior to the Status Assertion issuance datetime and it MUST end before
 the expiration datetime of the Digital Credential;
```

Comment:
Not all credentials expire.

I think we should provide examples for all cases of vc + sa with iat and exp.

There is also nbf which could interact negatively with iat and exp.

Reply:
I'd like to take a stab at addressing this complexity more explicitly, but I will save that for another PR

Reply:
good point. Issue here: https://github.com/peppelinux/draft-demarco-oauth-status-assertions/issues/71